bundles/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/handler/SatelEventLogHandler.java

```diff
@@ -192,13 +279,13 @@ private void readEvent(int eventIndex) {
                             + (readEventCmd.getObject() * 32 + readEventCmd.getUserControlNumber());
                     Optional<EventDescription> eventDescNext = getEventDescription(readEventCmd.getNextIndex());
```

Comment:
The `Optional` was here before, so ok for me, but I think you gain nothing by using it here. A simple null-check would be sufficient (in fact you are not doing anything other than that here) and produce less overhead.

Reply:
The method is used in other place as well (L205). Here, you're right, it does not makes to use it.